framework/web/migrations/m160313_153426_session_init.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+use yii\base\InvalidConfigException;
+use yii\web\DbSession;
+use yii\db\Migration;
+
+/**
+ * Initializes Session tables
+ *
+ * @author Misbahul D Munir <misbahuldmunir@gmail.com>
+ * @since 2.0.8
+ */
+class m160313_153426_session_init extends Migration
+{
+
+    /**
+     * @throws yii\base\InvalidConfigException
+     * @return DbSession
+     */
+    protected function getSession()
```

Comment:
This is bad idea: in this way migration depends on application configuration, which may change in the future.
If you switch to the memcache based session in the future, reverting this migration will produce PHP fatal error.